htdocs/core/lib/date.lib.php

```diff
@@ -852,7 +852,7 @@ function num_public_holiday($timestampStart, $timestampEnd, $country_code = '',
  */
 function num_between_day($timestampStart, $timestampEnd, $lastday = 0)
 {
-	if ($timestampStart < $timestampEnd)
+	if ($timestampStart <= $timestampEnd)
```

Comment:
When fixing a bug in old version, it is better to backport the fix from a recent version.
Can you instead suggest a PR that take the code of develop branch that seems to already have this fixed ?

Reply:
Hi @eldy 
Sorry but I did not find the fix in the latest develop version but maybe I did not look in the right version. (See my screenshot)
This is why then I parsed all the versions and I stop at 12.
<img width="1147" alt="LatestDevNum_between_day" src="https://github.com/user-attachments/assets/edaae5ca-5172-4b89-a660-14ac472c29c2">

Reply:
Hi @eldy 
The correction in develop seems to be wrong as we need to get :
0 if $lastday = 0
1 if $lastday = 1
and not always 0